tests/test_simpleButler.py

```diff
@@ -767,5 +757,68 @@ def test_skypix_templates(self):
         self.assertEqual(path, "test/warp/HSCA02713600_12345_12345_12345_12345")
 
 
+class DirectSimpleButlerTestCase(SimpleButlerTests, unittest.TestCase):
+    """Run tests against DirectButler implementation."""
+
+    datasetsManager = (
```

Comment:
I think this is the only one we use now and is the default so in some future clean up we can drop this manager override completely.